dask_ml/wrappers.py

```diff
@@ -323,6 +323,11 @@ class Incremental(ParallelPostFit):
            a single NumPy array, which may exhaust the memory of your worker.
            You probably want to always specify `scoring`.
 
+    max_iter : int, default 1
```

Comment:
The use of `max` in the name is strange to me here.  It looks like we always choose this number.  Perhaps this is a convention though?